Mage/src/main/java/mage/abilities/effects/common/continuous/BecomePermanentFacedownEffect.java

```diff
@@ -0,0 +1,73 @@
+package mage.abilities.effects.common.continuous;
+
+import mage.MageObjectReference;
+import mage.abilities.Ability;
+import mage.abilities.effects.ContinuousEffectImpl;
+import mage.cards.Cards;
+import mage.constants.Duration;
+import mage.constants.Layer;
+import mage.constants.Outcome;
+import mage.constants.SubLayer;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.target.targetpointer.FixedTargets;
+
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/**
+ * @author TheElk801
+ */
+public class BecomePermanentFacedownEffect extends ContinuousEffectImpl {
```

Comment:
Whole face down code is outdated in that cards. It must extends BecomesFaceDownCreatureEffect now or call BecomesFaceDownCreatureEffect.makeFaceDownObject